src/styles/less/global.less

```diff
@@ -1 +1 @@
-@import url('https://fonts.googleapis.com/css2?family=Bangers&family=Nunito:ital,wght@0,200;0,300;0,400;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bangers&family=Nunito&family=Mulish&family=Open+Sans:ital,wght@0,200;0,300;0,400;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,600;1,700;1,800;1,900&display=swap');
```

Comment:
According to the Figma doc, we'll need these fonts for the buttons and headers of our app.
Mulish for buttons!